src/nashpy/algorithms/lemke_howson_lex.py

```diff
@@ -0,0 +1,174 @@
+"""A class for the Lemke Howson algorithm"""
+import warnings
+from itertools import cycle
+
+import numpy as np
+
+from nashpy.integer_pivoting import (
+    make_tableau,
+    non_basic_variables,
+    zero_basic_variables,
+    pivot_tableau_lex,
+)
+
+
+def shift_tableau(tableau, shape):
+    """
+	Shift a tableau to ensure labels of pairs of tableaux coincide
+
+	Parameters
+	----------
+
+		tableau: a numpy array
+		shape: a tuple
+
+	Returns
+	-------
+
+		tableau: a numpy array
+	"""
+    return np.append(
+        np.roll(tableau[:, :-1], shape[0], axis=1),
+        np.ones((shape[0], 1)),
+        axis=1,
+    )
+
+
+def tableau_to_strategy_lex(tableau, basic_labels, strategy_labels):
```

Comment:
I don't think this differs from `lemke_howson.table_to_strategy` so let's import it from there and delete this one (and rename in the places it's used).